docs/ui/roles.md

```diff
@@ -0,0 +1,138 @@
+
+Custom roles created using an inherited role will follow Prefect's default workspace role permission curation as new functionality becomes available.
+
+To configure an inherited role when configuring a custom role, select the **Inherit permission from a default role** check box, then select the role from which the new role should inherit permissions.
+
+![Creating a custom role for a workspace using inherited permissions in Prefect Cloud](/img/ui/org-inherited-role.png)
+
+## Workspace role permissions
+
+The following permissions are available for custom roles.
+
+### Blocks
+
+| Permission | Description |
+| --- | --- |
+| View blocks | User can see configured blocks within a workspace. |
+| Create, edit, and delete blocks | User can create, edit, and delete blocks within a workspace. Includes permissions of **View blocks**. |
+
+### Task run concurrency
+
+| Permission | Description |
+| --- | --- |
+| View concurrency limits | User can see configured task run concurrency limits within a workspace. |		
+| Create, edit, and delete concurrency limits | User can create, edit, and delete task run concurrency limits within a workspace. Includes permissions of **View concurrency limits**. |		
+
+### Deployments
+
+| Permission | Description |
+| --- | --- |
+| View deployments | User can see configured deployments within a workspace. |
+| Run deployments | User can run deployments within a workspace. This does not give a user permission to execute the flow associated with the deployment. This only gives a user (via their key) the ability to run a deployment &mdash; another user/key must actually execute that flow, such as a service account with an appropriate role. Includes permissions of **View deployments**. |
+| Create and edit deployments | User can create and edit deployments within a workspace. Includes permissions of **View deployments** and **Run a deployment**. |
+| Create, edit, and delete deployments | User can create, edit, and delete deployments within a workspace. Includes permissions of **View deployments**, **Run deployments**, and **Create and edit deployments**. |
+
+### Flows
+
+| Permission | Description |
+| --- | --- |
+| View flows and flow runs  | User can see flows and flow runs within a workspace. |
+| Create, update, and run flows | User can create, update, and run flows within a workspace. Includes permissions of **View flows and flow runs**. |
+
+### Notifications
+
+| Permission | Description |
+| --- | --- |
+| View notification policies | User can see notification policies configured within a workspace. |
+| Create and edit notification policies | User can create and edit notification policies configured within a workspace. Includes permissions of **View notification policies**. |
+| Create, edit, and delete notification policies | User can create, edit, and delete notification policies configured within a workspace. Includes permissions of **View notification policies**. |
+
+### Search
```

Comment:
This permission section is being removed as part of https://github.com/PrefectHQ/nebula/issues/2030

View saved search filters is being removed as an independent scope. That functionality will be dependent on, "View flows and flow runs", but I don't think we need to call that out explicitly anywhere.

"Create, update, and delete saved search filters" will become part of the "Flows" section, dependent on the "View flows and flow runs" scope.